values/src/test/java/org/spine3/time/LocalTimesShould.java

```diff
@@ -53,34 +53,246 @@ public void obtain_current_LocalTime() {
     }
 
     @Test
+    public void obtain_LocalTime_using_hours_minutes() {
+        final int hours = 5;
+        final int minutes = 30;
+        final LocalTime localTime = LocalTimes.of(5, 30);
+        final Calendar calendar = Calendar.getInstance();
+        calendar.set(Calendar.HOUR, hours);
+        calendar.set(Calendar.MINUTE, minutes);
+
+        assertEquals(calendar.get(Calendar.HOUR), localTime.getHours());
+        assertEquals(calendar.get(Calendar.MINUTE), localTime.getMinutes());
+    }
+
+    @Test
+    public void obtain_LocalTime_using_hours_minutes_seconds() {
+        final int hours = 5;
+        final int minutes = 31;
+        final int seconds = 25;
+        final LocalTime localTime = LocalTimes.of(5, 31, 25);
+        final Calendar calendar = Calendar.getInstance();
+        calendar.set(Calendar.HOUR, hours);
+        calendar.set(Calendar.MINUTE, minutes);
+        calendar.set(Calendar.SECOND, seconds);
+
+        assertEquals(calendar.get(Calendar.HOUR), localTime.getHours());
+        assertEquals(calendar.get(Calendar.MINUTE), localTime.getMinutes());
+        assertEquals(calendar.get(Calendar.SECOND), localTime.getSeconds());
+    }
+
+    @Test
+    public void obtain_LocalTime_using_hours_minutes_seconds_millis() {
+        final int hours = 5;
+        final int minutes = 31;
+        final int seconds = 25;
+        final int millis = 125;
+        final LocalTime localTime = LocalTimes.of(5, 31, 25, 125);
+        final Calendar calendar = Calendar.getInstance();
+        calendar.set(Calendar.HOUR, hours);
+        calendar.set(Calendar.MINUTE, minutes);
+        calendar.set(Calendar.SECOND, seconds);
+        calendar.set(Calendar.MILLISECOND, millis);
+
+        assertEquals(calendar.get(Calendar.HOUR), localTime.getHours());
+        assertEquals(calendar.get(Calendar.MINUTE), localTime.getMinutes());
+        assertEquals(calendar.get(Calendar.SECOND), localTime.getSeconds());
+        assertEquals(calendar.get(Calendar.MILLISECOND), localTime.getMillis());
+    }
+
+    @Test
+    public void obtain_LocalTime_using_hours_minutes_seconds_millis_nanos() {
+        final int hours = 5;
+        final int minutes = 31;
+        final int seconds = 25;
+        final int millis = 125;
+        final long nanos = 2356L;
+        final LocalTime localTime = LocalTimes.of(5, 31, 25, 125, 2356L);
+        final Calendar calendar = Calendar.getInstance();
```

Comment:
Please have static factory methods for calendars that take 2, 3, and 4 arguments. Consider placing them into a class `Calendars` that in the same package with `Tests`.